flyteadmin_config.yaml

```diff
@@ -200,3 +200,5 @@ qualityOfService:
     development: LOW
     staging: MEDIUM
     # by default production has an UNDEFINED tier when it is omitted from the configuration
+namespace_mapping:
+   template: "{{ project }}-{{ domain }}" # Default namespace mapping template.
```

Comment:
How about adding the prefix to the default template which defaults to an empty string?
"{{prefix}}{{ project }}-{{ domain }}"

& add project-prefix as an additional cmd line option to the project & file registration

Reply:
if the prefix isn't dynamic, why can't we statically define it in here?
```
tempate: "flyteatjpm-{{ project }}-{{ domain }}"
```

Reply:
resolved offline - we'll use static prefixes